src/popup/pages/RestoreAccount/index.js

```diff
@@ -45,6 +45,7 @@ class RestoreAccount extends React.Component {
       Toast.info(getLanguage('seedLengthError'))
       return
     }
+    console.log(`mnemonic is "${mnemonic}"`)
     let mnemonicValid = bip39.validateMnemonic(mnemonic)
```

Comment:
Putting debugger here and going into validateMnemonic https://github.com/bitcoinjs/bip39/blob/200c6a9/src/index.js#L148-L153
```js
function validateMnemonic(mnemonic, wordlist) {
    try {
        mnemonicToEntropy(mnemonic, wordlist);
    }
    catch (e) {
        return false;
```

`e` in catch is
```
TypeError: Cannot read properties of undefined (reading 'call')
    at Hash.CipherBase (popup.js:39643)
    at new Hash (popup.js:42098)
    at createHash (popup.js:42118)
    at deriveChecksumBits (popup.js:37288)
    at mnemonicToEntropy (popup.js:37345)
    at Object.validateMnemonic (popup.js:37394)
    at popup.js:26354
    at HTMLUnknownElement.callCallback (popup.js:62273)
    at Object.invokeGuardedCallbackDev (popup.js:62322)
    at invokeGuardedCallback (popup.js:62384)
```

that line comes from
https://github.com/crypto-browserify/cipher-base/blob/bc18dc84/index.js#L7
```js
var Transform = require('stream').Transform
..
function CipherBase (hashMode) {
  Transform.call(this)
```

It seems `stream` is not polyfilled correctly. It only has `require('stream').Stream`

Reply:
Compiled code before and after
```diff
- var Transform = __webpack_require__(/*! stream */ "./node_modules/stream-browserify/index.js").Transform
+ var Transform = __webpack_require__(/*! stream */ "./node_modules/stream/index.js").Transform
```

Reply:
Hey, seems like Oasis is leaving webpack world, but you could try sth like this https://github.com/ysz/oasis-wallet-ext/compare/upgrade-webpack...buberdds:upgrade-webpack 

There is also a new PR in `cipher-base` https://github.com/crypto-browserify/cipher-base/pull/16 which targets that issue.